src/languages/en.js

```diff
@@ -521,6 +524,18 @@ export default {
         phrase5: 'Money transmission is provided by Expensify Payments LLC (NMLS ID:2017010) pursuant to its',
         phrase6: 'licenses',
     },
+    validateCodeForm: {
+        magicCodeNotReceived: "Didn't receive a magic code?",
+        fillTwoFactorAuthOrRecoveryCode: 'Please enter your two factor code or recovery code',
+        enterTwoFactorOrRecoveryCode: 'Enter your two factor code or recovery code',
+        twoFactorCode: 'Two factor code',
+        requiredWhen2FAEnabled: 'Required when 2FA is enabled',
+        error: {
+            pleaseFillMagicCode: 'Please enter your magic code',
+            incorrectMagicCode: 'Incorrect magic code. Please try again.',
```

Comment:
NAB: We don't say `Please try again.` on web (though maybe we should just to be polite 😅) 

![image](https://user-images.githubusercontent.com/16747822/215635818-ef4f47bd-786d-472a-8b3b-d34297bc937b.png)

Reply:
Niice I'll make this match olddot - who needs to be polite 😅